devolving_music/models/song_comparison.py

```diff
@@ -12,6 +11,8 @@ class SongComparison(models.Model):
 
     @staticmethod
     def get_event_comparisons(Event):
-        comparison_event_submissions = list(sub for sub in SongComparison.objects.select_related('first_submission').filter(first_submission__event__exact=Event).order_by('created_at'))
+        comparison_event_submissions = list(sub for sub in \
```

Comment:
Oh, I missed this in my earlier review. `sub for sub in` doesn't do anything here, take it out.